nadlogar/problems/forms.py

```diff
@@ -1,59 +1,40 @@
 from django import forms
 from django.core.exceptions import ValidationError
-from django.forms.models import ModelChoiceIterator
-
-from .models import ProblemText
 
 
 def problem_form(content_type, *args, **kwargs):
     Generator = content_type.model_class()
-    example_problem = kwargs.get("instance", Generator())
-    example_data = example_problem.example_data()
-
-    class ProblemTextChoiceIterator(ModelChoiceIterator):
-        def choice(self, obj):
-            return (self.field.prepare_value(obj), obj.render(example_data))
 
     class ProblemForm(forms.ModelForm):
-        instruction = forms.CharField(
```

Comment:
Če prav razumem, je ta razred nek boolean field, ki glede na to, ali imamo privzeto besedilo ali ne, pridobi ustrezne atribute.

Reply:
Ja, to je dodatni atribut, ki ga ni v modelu, ampak je samo v obrazcu in se nastavi glede na to, ali uporabnik klikne na izbiro (radio button) pri privzetem ali lastnem besedilu. Podatke pa shranimo tako, da sta `instruction` in `solution` pri privzetem besedilu prazna, pri lastnem besedilu pa ne.